src/service/service-directory-api/src/FamilyHubs.ServiceDirectory.Data/Entities/Staging/ServicesTemp.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace FamilyHubs.ServiceDirectory.Data.Entities.Staging;
+
+public class ServicesTemp
+{
+    public long Id { get; set; }

```

Comment:
I think the ID here is either a GUID or String (dunno if it results in any performance difference, technically they are Guid's from ORs perspective), as the ID is supposed to be the same as the incoming service's ID e.g., "ac148810-d857-441c-9679-408f346de14b" from the OR example

Reply:
Fixed